### PR TITLE
Allow overriding passenger_group in apache::vhost

### PR DESCRIPTION
**Pull request by smortex**
Just like we can use `passenger_user` in a VHost, allow setting `passenger_group`.

### CODE REVIEW

**Comment by david22swan** [comment]
@smortex Please alter the docs to reflect your changes and add some test coverage.

**Comment by smortex** [comment]
Working on it.  The test suite lacks test for the `passenger_*` params… Adding them.

**Comment by smortex** [comment]
Done!  This helped finding an unrelated problem in the passenger template 😉

**Comment by smortex** [comment]
@david22swan maybe the *needs-docs* and *needs-tests* labels can be removed?

File: templates/vhost/_passenger.erb [hunk 1]
```diff
@@ -25,11 +25,14 @@
...
 <% end -%>
+<% if @passenger_group -%>
+  PassengerGroup <%= @passenger_group %>
+<% end -%>
 <% if @passenger_high_performance -%>
   PassengerHighPerformance <%= scope.call_function('apache::bool2httpd', [@passenger_high_performance]) %>
 <% end -%>
 <% if @passenger_nodejs -%>
-  PassengerNodejs <%= @passenger_nodejs -%>
+  PassengerNodejs <%= @passenger_nodejs %>
```
Comment (eputnam) [inline templates/vhost/_passenger.erb]: nice catch! i hope the config parser is hardy enough to handle a newline :)

Comment (smortex) [reply templates/vhost/_passenger.erb]: I don't think so 😉 

```
PassengerNodejs takes one argument, The Node.js command to use.
```

Comment (eputnam) [reply templates/vhost/_passenger.erb]: ah ok, so we def need that newline

**Pull request merged**